attributes.go

```diff
@@ -108,3 +110,25 @@ func statusCodeAttribute(protocol string, serverErr error) attribute.KeyValue {
 	}
 	return codeKey.String("success")
 }
+
+func headerAttributes(protocol, eventType string, metadata http.Header, allowedKeys []string) []attribute.KeyValue {
+	var attributes []attribute.KeyValue
```

Comment:
Let's size this: `attributes := make([]attribute.KeyValue, 0, len(allowedKeys))`

Reply:
resolved in 39ab387349f28a2519527f3565e649a7e06e4380